### PR TITLE
feat: add ofiuco@0.3.7 and freeze rules_poetry at v0.3.6

### PR DESCRIPTION
**Pull request by oxidase**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (rules_ophiuchus) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (ofiuco) have been updated in this PR. Please review the changes.

File: modules/ofiuco/0.3.7/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+matrix:
+  platform:
+    - centos7
+    - debian10
+    - ubuntu2004
+    - macos
+  bazel:
+    - 7.x
```
Comment (meteorcloudy) [inline modules/ofiuco/0.3.7/presubmit.yml:8]: nit: Does this module support 6.x? If not, you can also put `bazel_compatibility = [">=7.0.0"]` in your MODULE.bazel file. See https://bazel.build/rules/lib/globals/module#parameters_5

Comment (oxidase) [reply modules/ofiuco/0.3.7/presubmit.yml:8]: @meteorcloudy  added `bazel_compatibility = [">=7.0.0"]`

**Review by meteorcloudy — APPROVED** [review]
Thanks!

File: modules/ofiuco/0.3.7/presubmit.yml (hunk 1 above)
Comment (meteorcloudy) [reply modules/ofiuco/0.3.7/presubmit.yml:8]: Thanks!

**Pull request merged**